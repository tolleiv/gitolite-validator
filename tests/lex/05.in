repo project/(a|b|c)/([a-zA-Z0-9_\.\-]+/)*[a-zA-Z0-9_\.\-]+
    C       =   @devs
    RW+     =   @users
    config gitweb.url = git@git.example.org:%GL_REPO
