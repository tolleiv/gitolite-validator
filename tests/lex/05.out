{t_string[1] repo}
{t_string[1] project/(a|b|c)/([a-zA-Z0-9_\.\-]+/)*[a-zA-Z0-9_\.\-]+}
{t_eol[1] }
{t_string[2] C}
{t_assign[2] }
{t_group[2] devs}
{t_eol[2] }
{t_string[3] RW+}
{t_assign[3] }
{t_group[3] users}
{t_eol[3] }
{t_string[4] config}
{t_string[4] gitweb.url}
{t_assign[4] }
{t_string[4] git@git.example.org:%GL_REPO}
{t_eol[4] }
